docker/raspberry_pi.md

```diff
@@ -41,34 +41,28 @@ sudo add-apt-repository \
 sudo apt-get install -y docker-ce
 ```
 
-As the docker binary for Ubuntu 19.10 (Eoan) is not provided, it is necessary to install the docker binary
-for Ubuntu 18.04 LTS (Bionic). You should replace `$(lsb_release -cs)` with `bionic` in `add-apt-repository` command.
-
 The details to install Docker are [here](https://docs.docker.com/install/linux/docker-ce/ubuntu/).
 
 ### Docker compose
 
 The Docker Compose binary for aarch64 is not provided. It is necessary to build it from its source code.
-You can install the docker compose version 1.25.1 by running the commands as shown:
+You can install the docker compose version 1.27.4 by running the commands as shown:
 
 ```
-git clone -b 1.25.1 https://github.com/docker/compose.git
+git clone -b 1.27.4 https://github.com/docker/compose.git
 cd compose/
-sed -i -e "43i'setuptools<45.0.0'," setup.py
 sudo ./script/build/linux
 sudo cp dist/docker-compose-Linux-aarch64 /usr/local/bin/docker-compose
 ```
 
-Note: A patch to avoid a build error is added by using the sed command.
-
 ## How to build Orion
 
 To build the docker images of Orion, Clone the Orion repository and run the `docker build` command.
 
 ```
 git clone https://github.com/telefonicaid/fiware-orion.git
 cd fiware-orion/docker
-docker build --build-arg=centos7 -t orion .
+docker build -t orion .
```

Comment:
In the case of CentOS 8, there are docker images with the same tag `centos8.3.2011` for amd64 and aarch64. So the procedure  to build a docker image is the same.

https://hub.docker.com/layers/centos/library/centos/centos8.3.2011/images/sha256-7723d6b5d15b1c64d0a82ee6298c66cf8c27179e1c8a458e719041ffd08cd091?context=explore

Reply:
Ok. Nice.

NTC